app/controllers/api/solutions_controller.rb

```diff
@@ -5,8 +5,12 @@ def index
         current_user,
         criteria: params[:criteria],
         status: params[:status],
-        mentoring_status: params[:mentoring_status]
+        mentoring_status: params[:mentoring_status],
+        page: params[:page],
+        per: params[:per],
+        sort: params[:sort]
```

Comment:
@kntsoriano Could we rename this param to `order` in the JS please? (I'm doing it in Ruby as part of a commit I'm adding to this)